src/tritonserver.cc

```diff
@@ -2933,6 +2935,35 @@ TRITONSERVER_ServerModelStatistics(
           metadata, inference_stats, "cache_miss",
           infer_stats.cache_miss_count_, infer_stats.cache_miss_duration_ns_);
 
+      // Add response statistics
+      triton::common::TritonJson::Value response_stats(
```

Comment:
Can you add a sample output of stats endpoint including these new stats to the PR description?

Reply:
Sure.